### PR TITLE
feat: send message attributes to NR

### PR DESCRIPTION
**Pull request by sloria**
Part of https://app.asana.com/0/1159194223099561/1171415409585889/f

This will allow us to access the "SentTimestamp" attribute from NR.

cc @nate-selzer

### CODE REVIEW

File: tqp/newrelic.py [hunk 1]
```diff
@@ -27,6 +27,9 @@ def _hook_send_newrelic_event(self, payload, success):
                 "topic": payload["topic"],
                 "queue_name": self.queue_name,
                 "success": str(success),
+                "attributes": payload["meta"].get("attributes", {})
+                if payload["meta"]
+                else {},
```
Comment (taion) [inline tqp/newrelic.py:32]:
```suggestion
                "attributes": (payload["meta"].get("attributes", {})
                if payload["meta"]
                else {}),
```

so black doesn't screw this up

File: tqp/topic_queue_poller.py [hunk 2]
```diff
@@ -138,12 +138,17 @@ def get_message_payload(self, msg):
...
             "handler": handler,
             "message": message,
-            "meta": {"body": body, "topic": topic[len(self.prefix) :],}
+            "meta": {
+                "body": body,
+                "topic": topic[len(self.prefix) :],
+                "attributes": attributes,
+            }
             if with_meta
             else None,
```
Comment (taion) [inline tqp/topic_queue_poller.py:153]:
```suggestion
            "meta": ({
                "body": body,
                "topic": topic[len(self.prefix) :],
                "attributes": attributes,
            }
            if with_meta
            else None),
```

File: tqp/newrelic.py [hunk 3]
```diff
@@ -27,6 +27,9 @@ def _hook_send_newrelic_event(self, payload, success):
                 "topic": payload["topic"],
                 "queue_name": self.queue_name,
                 "success": str(success),
+                "attributes": payload["meta"].get("attributes", {})
```
Comment (itajaja) [inline tqp/newrelic.py:30]: looks like `meta` is always set

Comment (taion) [reply tqp/newrelic.py:30]: only when `with_meta` is set: https://github.com/4Catalyzer/tqp/pull/23/files#diff-ba5945493e7a28bddde9aa9f54dd499fR152

File: tqp/topic_queue_poller.py [hunk 4]
```diff
@@ -138,14 +138,18 @@ def get_message_payload(self, msg):
...
+        attributes = msg.Attributes
 
         return {
             "topic": topic,
             "handler": handler,
             "message": message,
-            "meta": {"body": body, "topic": topic[len(self.prefix) :],}
-            if with_meta
-            else None,
+            "attribute": attributes,
```
Comment (r1b) [inline tqp/topic_queue_poller.py:147]: `"attributes": attributes,` ?

**Pull request merged**